src/sensor/ping360.h

```diff
@@ -358,8 +420,11 @@ class Ping360 : public PingSensor
      */
 ///@{
 
+    // TODO: static members should be defined in .cpp
     // firmware constants
     static const uint16_t _firmwareMaxNumberOfPoints = 1200;
+    const uint16_t _firmwareMaxTransmitDuration = 500;
+    const uint16_t _firmwareMinTransmitDuration = 5;
```

Comment:
static const

Reply:
if this is private, I would move it to the .cpp file and use constexpr in an unamed namespace.

Reply:
Check the TODO added above. This is an existing issue.

Reply:
I will move them to anonymous namespace in cpp.

Reply:
we can't do anonymous namespace because we have code using the members in .h. I moved the definitions to .cpp, they are still class members, `static const`.